### PR TITLE
Bugfix/qml warnings

### LINKED ISSUE #3070: Fix QML warnings
See #3069

### PR DESCRIPTION
**Pull request by camilasan**
Partially fix #3070.
Most of these warnings were related to the User Status feature.
There are more PR's coming to fix some of these warning in other parts of the QML code.

### CODE REVIEW

File: src/gui/tray/Window.qml [hunk 1]
```diff
@@ -21,9 +21,6 @@ Window {
 
     readonly property int maxMenuHeight: Style.trayWindowHeight - Style.trayWindowHeaderHeight - 2 * Style.trayWindowBorderWidth
 
-    Accessible.role: Accessible.Application
-    Accessible.name: qsTr("Nextcloud desktop main dialog")
```
Comment (allexzander) [inline src/gui/tray/Window.qml:25]: @camilasan Is this really not needed anymore?

Comment (camilasan) [reply src/gui/tray/Window.qml:25]: This was throwing a warning: Accessible must be attached to an Item. And look at lines 90 and 91: the exact same Accessible was already defined there.

**Review by allexzander — APPROVED** [review]

File: src/gui/tray/UserLine.qml [hunk 2]
```diff
@@ -99,27 +99,23 @@ MenuItem {
...
+                            Layout.alignment: Qt.AlignLeft | Qt.AlignBottom
                             width: 128
                             text: name
                             elide: Text.ElideRight
                             color: "black"
                             font.pixelSize: Style.topLinePixelSize
                             font.bold: true
                         }
-                        Row {
+                        RowLayout {
```
[the remark this answers is not in the document]
Comment (camilasan) [reply src/gui/tray/UserLine.qml]: I tried to go back to Row but I found it harder to keep Linux and Windows the same when comes to aligning the emoji with the message.

[the remark this answers is not in the document]
Comment (mgallien) [reply src/gui/tray/UserLine.qml:113]: my question was more about getting rid of the parent element and just using the parent RowLayout item
maybe it is just too hard to fix alignment issues in that case but I am wondering if that is possible (and that would reduce complexity)

[the remark this answers is not in the document]
Comment (camilasan) [reply src/gui/tray/UserLine.qml]: Please check my last commit. let me know if that is better or worst...

[the remark this answers is not in the document]
Comment (mgallien) [reply src/gui/tray/UserLine.qml]: they are looking fine for me

**Review by mgallien — APPROVED** [review]

**Pull request merged**